crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -36,6 +39,10 @@ pub struct Create {
     /// Custom open zeppelin contract class hash of declared contract
     #[clap(short, long)]
     pub class_hash: Option<String>,
+
+    /// If passed, create account from keystore and starkli account JSON file
+    #[clap(short, long)]
+    pub from_keystore: bool,
```

Comment:
There already is a `--keystore` common flag, wouldn't it be confusing?

Reply:
We could re-use it for exactly this purpose right? if `--keystore` was passed and `--account` is a path, we know we import starkli account; otherwise it is standard oz account

Reply:
I guess we should take under account also this thread https://github.com/foundry-rs/starknet-foundry/pull/634/files#r1322706719

If we put everything into one file, I don't think we should alter `--account` to be a path in case of starkli account, but rather an account name (as it is right now). If that's the case, only `--keystore` flag would indicate that we're using starkli account.